packages/dds/cell/src/cell.ts

```diff
@@ -158,23 +219,40 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>> impl
         return this.data === undefined;
     }
 
+    /**
+     * {@inheritDoc ISharedCell.getAttribution}
+     * @alpha
+     */
+    public getAttribution(): AttributionKey | undefined {
+        return this.attribution;
+    }
+
+    /**
+     * {@inheritDoc ISharedCell.setAttribution}
+     * @alpha
```

Comment:
no need to mark this @alpha as it's private. also there isn't any doc on the interface (since it was deleted). i'd just remove the doc comment, it's pretty clear what this is doing based on impl.